### ISSUE TITLE
A11y_MicrosoftDynamics365ImportTool_ModelDetailsPropertiesTab_Screenreader: NVDA does not read any non-interactive element using up/down arrow key on the screen.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [nvaccess/nvda#9366]
Application Name: Microsoft Dynamics 365 Import Tool.

### Steps to reproduce:
1.Launch the Microsoft Dynamics 365 Import Tool (Preview).
2Hit enter on any of the model->Navigate to Properties tab
3.Start NVDA.

### Actual behavior:
NVDA does not read any non-interactive element present on the screen using up/down arrow key in both Properties and Summary tab.

### Expected behavior:
NVDA should read each and every information available on the screen whether they are interactive or non-interactive.

#### Have you tried any other versions of NVDA? Yes
![20060463_Properties_NVDA](https://user-images.githubusercontent.com/39204909/54107005-e9371980-43fd-11e9-84bd-3f30d6ab161a.png)

**Comment by Adriani90** [comment]
@AccessibilityTestingTeam-TCS this is really hard to reproduce for everyone. Could you please report on how it is now with NVDA 2020.1?
Note for this to reproduce it seems you need an abonament. Not everyone has an abonament for Dynamics 365.
I suggest you to reach to NV Access directly when having such specific issues since this is not very trivial to investigate.

**Comment by ehollig** [comment]
As @Adriani90 commented in https://github.com/nvaccess/nvda/issues/9366#issuecomment-628902778 requesting for more information, and we have not heard from @AccessibilityTestingTeam-TCS in several years, I'm going to close this issue. Please reach out to NvAccess directly if you have additional concerns about this.
